### PR TITLE
Rename instance attachment to match versions 23 and earlier

### PR DESCRIPTION
**Pull request by cygnus8595**
In versions 23 and earlier, Apigee instance attachments were named <region>-<environment>. Later versions swapped that to <environment>-<region>, which will cause downtime when upgrading to the latest module as the attachments would have  to be destroyed and re-created.

This PR just swaps the names back to how they were to avoid causing downtime for just a name.
---
**Checklist**

I applicable, I acknowledge that I have:
- [X] Read the [contributing guide](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md)
- [X] Ran `terraform fmt` on all modified files
- [X] Regenerated the relevant README.md files using [`tools/tfdoc.py`](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md#fabric-tools)
- [X] Made sure all relevant tests pass

### CODE REVIEW

File: modules/apigee/variables.tf [hunk 1]
```diff
@@ -66,11 +66,13 @@ variable "instances" {
   type = map(object({
     name                          = optional(string)
     display_name                  = optional(string)
+    name                          = optional(string)
     description                   = optional(string, "Terraform-managed")
     runtime_ip_cidr_range         = optional(string)
     troubleshooting_ip_cidr_range = optional(string)
     disk_encryption_key           = optional(string)
     consumer_accept_list          = optional(list(string))
+    environments                  = optional(list(string))
```
Comment (juliocc) [inline modules/apigee/variables.tf:75]: Duplicate

File: modules/apigee/variables.tf [hunk 2]
```diff
@@ -66,11 +66,13 @@ variable "instances" {
   type = map(object({
     name                          = optional(string)
     display_name                  = optional(string)
+    name                          = optional(string)
```
Comment (juliocc) [inline modules/apigee/variables.tf:69]: Duplicate

**Review by juliocc — DISMISSED** [review]
@apichick can you take a look?

File: modules/apigee/variables.tf (hunk 2 above)
Comment (cygnus8595) [reply modules/apigee/variables.tf:69]: Sorry, must have got duplicated in a merge. I was working on this simultaneously.

File: modules/apigee/variables.tf (hunk 1 above)
Comment (cygnus8595) [reply modules/apigee/variables.tf:75]: Same as above, merge issue.

**Review by apichick — APPROVED** [review]

**Pull request merged**